src/makeCalculator.js

```diff
@@ -4,7 +4,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
```

Comment:
[CODE KNOWLEDGE] - There is no need to take two parameters for the `add` method. It should only take one parameter and add it to the current `result`.